ocaml/tests/test_repository_helpers.ml

```diff
@@ -1304,7 +1322,8 @@ module GetRpmUpdateInJson = Generic.MakeStateless (struct
             ),
             "libpath-utils.noarch    0.2.2-1.el7      epel" (* repository name is "epel" *)
           ),
-          None
+          Error Api_errors.(Server_error(
+              internal_error, ["Found update from un-managed repository"]))
```

Comment:
```suggestion
              internal_error, ["Found update from unmanaged repository"]))
```

if the repository is invalid, maybe you want to use an `invalid_value` error, rather than internal error?